assets/stylesheets/kitten/components/form/_text-input-with-limit.scss

```diff
@@ -63,6 +57,10 @@
                  -1px 1px 0 #fff;
     pointer-events: none;
 
+    &.is-disabled {
```

Comment:
Je pensais plutôt soit :
`.k-TextInputLimit.is-disabled & {` => `.k-TextInputLimit.is-disabled .k-TextInputLimit__counter`

soit :
`.k-TextInputLimit &.is-disabled {` => `.k-TextInputLimit .k-TextInputLimit__counter.is-disabled`